### PR TITLE
[flink] add table comment for create table in sync action

### PR DESCRIPTION
**Pull request by zhangjun0x01**
### Purpose

in flink sync action , sync the table comment when create new table.

### CODE REVIEW

File: paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/cdc/mysql/MySqlActionUtils.java [hunk 1]
```diff
@@ -79,13 +79,13 @@ static Connection getConnection(Configuration mySqlConfig, boolean tinyint1NotBo
             throws Exception {
         String url =
                 String.format(
-                        "jdbc:mysql://%s:%d%s",
+                        "jdbc:mysql://%s:%d?useInformationSchema=true%s",
```
Comment (JingsongLi) [inline paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/cdc/mysql/MySqlActionUtils.java:82]: Should we use an option to open this feature?
I don't know how much impact it has in performance.

Comment (zhangjun0x01) [reply paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/cdc/mysql/MySqlActionUtils.java:82]: the param in default `true` in  mysql 8.0.3 ([doc](https://dev.mysql.com/doc/connector-j/8.1/en/connector-j-connp-props-metadata.html)) ,but it is default `false` in mysql    5.7,  if we add an option and when we set it to `false` ,do    we should set it to `false` explicitly in mysql8 ?

Comment (JingsongLi) [reply paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/cdc/mysql/MySqlActionUtils.java:82]: Default auto?

Comment (JingsongLi) [reply paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/cdc/mysql/MySqlActionUtils.java:82]: Or default we can just disable table comments.

**Comment by zhuangchong** [comment]
Already supported, I will close this PR first. If you have any questions, you can open it again.

**Pull request closed without merging**